data-collection/src/test/java/integration/canaryprojects/FluentIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package integration.canaryprojects;
+
+import integration.IntegrationBaseTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import refactoringml.db.ProcessMetrics;
+import refactoringml.db.RefactoringCommit;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class FluentIntegrationTest extends IntegrationBaseTest {
+	@Override
+	protected String getRepo() {
+		return "https://github.com/FluentLenium/FluentLenium.git";
+	}
+
+	@Test
+	public void moveSourceRefactoring(){
+		String moveSource = "85c68373dabe32334933bdf6e67091534fc1504a";
+		assertRefactoring(getRefactoringCommits(), moveSource, "Move Source Dir", 46);
+	}
+
+	/*@Test
```

Comment:
You want this test to be commented out?

Reply:
It was not used, but I will also add it.